test/test_helper.rb

```diff
@@ -6,9 +6,12 @@
 require 'rails'
 require 'action_controller/railtie'
 require 'mocha/mini_test'
+require 'pry-byebug' if ENV['PRY'] == '1'
```

Comment:
Baby nitpick: I don't think it's necessary to check against a string
```suggestion
require 'pry-byebug' if ENV['PRY']
```